orc8r/lib/go/service/service.go

```diff
@@ -119,7 +119,11 @@ func NewServiceWithOptions(moduleName string, serviceName string, serverOptions
 		StartTimeSecs:       uint64(time.Now().Unix()),
 		Config:              configMap,
 	}
-	protos.RegisterService303Server(service.GrpcServer, service)
+	if protected303Server {
```

Comment:
Better to use two methods. NewGatewayServiceWithOptions. NewOrc8rServiceWithOptions

Reply:
I implemented the methods as suggested